python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -949,6 +982,101 @@ def calculateMetrics(self, science, difference, diaSources, kernelSources):
                 raise BadSubtractionError(ratio=metrics.differenceFootprintRatioStdev,
                                           threshold=self.config.badSubtractionVariationThreshold)
 
+    def filterSatellites(self, diaSources, science):
+
+        wcs = science.getWcs()
+        nbbox = []
+
+        for source in diaSources:
+            fp = source.getFootprint()
+            source_bbox = fp.getBBox()
+
+            corners = [wcs.pixelToSky(source_bbox.beginX, source_bbox.beginY),
+                       wcs.pixelToSky(source_bbox.beginX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.endY),
+                       wcs.pixelToSky(source_bbox.endX, source_bbox.beginY)]
+
+            tmp = []
+            for c, corner in enumerate(corners):
+                tmp.append([corner.getRa().asDegrees(), corner.getDec().asDegrees()])
+            nbbox.append(tmp)
+
+        detector_id = science.getDetector().getId()
+        visit_id = science.getInfo().getVisitInfo().getId()
+
+        dia_sources_json = []
+        for i, source in enumerate(diaSources):
+            dia_sources_json.append(
+                {"diasource_id": source['id'], "bbox": nbbox[i]})
```

Comment:
```suggestion
        dia_sources_json = []
        for source in diaSources:
            source_bbox = source.getFootprint().getBBox()
            corners = [wcs.pixelToSky(Point2D(c)) for c in source_bbox.getCorners()]
            bbox_radec = [[pt.getRa().asDegrees(), pt.getDec().asDegrees()] for pt in corners]
            dia_sources_json.append({"diasource_id": source["id"], "bbox": bbox_radec})
```

Even if you don't go that compact, `for corner in corners:` is better than `for c, corner in enumerate(corners):` when you don't use `c`.